canonical-web-and-design/cn.ubuntu.com
# Remove unused, complex secondary nav system

## Done

Removed some old templates and template fragments relating to secondary nav that doesn't exist

## QA

- http://localhost:8010/about 
- click through all the pages and ensure that they load and look the same as live

## Issue / Card

Fixes #306
